decentraland/unity-renderer
# fix: global-scenes didn't have sdk7 in their serializable settings

## What does this PR change?
Close https://github.com/decentraland/sdk/issues/581 and add a test.

## How to test the changes?
Steps in the playground: 
1. Go to the playground with the next query params added: https://chore-support-dcl-explorer.sdk-playground.pages.dev/?&sdk-branch=chore/use-dcl-explorer&explorer-branch=fix/sdk7-global-scenes
2. When all is ready, click on Run and it should work
3. Check that is currently not working without this branch: https://chore-support-dcl-explorer.sdk-playground.pages.dev/?&sdk-branch=chore/use-dcl-explorer&explorer-branch=dev
